test/Posts/PostsTest.php

```diff
@@ -43,11 +43,12 @@ public function testLookup() {
     * @expectedException Traackr\MissingParameterException
```

Comment:
Since not exception is being raised, this directive probably needs to be removed.